### DISCUSSION TITLE
소중한 선물을 잊지 않게 해주는 기프티콘 관리, MyGift

### DISCUSSION THREAD

**Discussion by Dylan-yoon** [WB-GiftWallet/GiftWallet#34] [공지사항]
## 👋 Welcome!

안녕하세요 MyGift 개발진 Baem, Woong 입니다.
저희 서비스에 관심 가져주셔서 감사합니다!

`MyGift`는 현대 사회에서 많은 사람들이 주고 받는 기프티콘을 효율적으로 관리하고 잊지 않도록 도와주는 편리한 어플입니다. 
기프티콘은 자주 선물로 주고받는 형태이지만 종종 우리는 이러한 기프티콘을 잊어버리거나 유효기간이 지나버리는 경우가 생기곤 합니다.
MyGift은 이러한 문제를 해결하기 위해 설계되었습니다.

많은 관심 부탁드립니다.

**Comment by iOS-Woong** [comment]
# MyGift

개발기간 : 2023.03 ~ 2022.05  
개발과정노트 : [Notion - MyGift](https://glossy-modem-fe7.notion.site/Gift-Wallet-53d4eb1058854bac97d034ee1a71abc9)  
개발자문의: [오픈카톡](https://open.kakao.com/o/gjrFstkf)

[![appStore](https://user-images.githubusercontent.com/50910456/173174832-7d395623-ceb3-4796-b718-22e550af6934.svg)](https://apps.apple.com/kr/app/mygift/id6448777213)

## [About]

![그룹 38513](https://hackmd.io/_uploads/rJVlTezB2.png)

사용자의 기프티콘의 등록 및 관리하는 앱입니다


## [Update]

- version 1.0.0 : 
    -앱 스토어 출시 ( 23.05.13 )
  
- version 1.0.1 :
    -쿠폰 추가 시 브랜드, 만료일 자동입력 기능의 버그 수정
    -메인화면에서 유저 스크롤 가능 영역의 조절
    -기타 안정성 개선 (순환참조 및 등록시 미업데이트 에러해결)
    -마케팅URL 링크 추가 ( 23.05.23 )
